docs/api/en/constants/Renderer.html

```diff
@@ -56,6 +56,18 @@ <h2>Tone Mapping</h2>
 
 		</p>
 
+		<h2>Triangle draw Modes</h2>
+		<code>
+		THREE.TrianglesDrawMode
+		THREE.TriangleStripDrawMode
+		THREE.TriangleFanDrawMode
+		</code>
+		<p>
+		[page:constant TrianglesDrawMode] is the mode to draw a series of separate triangles. <th></th><br />
```

Comment:
Maybe we should not that the renderer only supports `THREE.TrianglesDrawMode`, see #18041.

We have kept `THREE.TriangleStripDrawMode` and `THREE.TriangleFanDrawMode` only for the usage of `BufferGeometryUtils.convertToTriangles()` (which is a utility method for converting geometries, see #18031).

Reply:
So these constants should only be documented in `BufferGeometry.toTrianglesDrawMode()` documentation, correct?

Reply:
That's probably the best choice. Otherwise users might think they can actually define the draw mode via those constants which is not supported.

Reply:
Closing the PR and fix the docs with a new one.